src/main/java/bogdanSandu/driver/BrowserManager.java

```diff
@@ -32,7 +35,15 @@ public static ChromeDriver getChromedriverForMobile(){
         return new ChromeDriver(chromeOptions);
     }
 
-    public static void closeBrowser(ChromeDriver driver){
+    public static RemoteWebDriver creatRemoteWebDriver(){
+        try {
+            return new RemoteWebDriver(new URL("http://localhost:4444/ui"), new ChromeOptions());
```

Comment:
Please change the url from "http://localhost:4444/ui" to "http://localhost:4444/". Thanks!